GA-worksheet.md

```diff
@@ -26,44 +31,58 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
+    You can its better in an orginization sense.
 
 2. What is normalize.css?
+    The CSS File
 
 3. Why do we need to add a padding property to the `<ul>` tag?
+    To add space between the links / text
 
 4. What does the “inline” property do to a list of items?
+    It puts them in a straight line.
 
 5. What do the following padding values do?
+    
+
 
     padding: 25px 50px 75px 100px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is 25px
+    * right padding is 50px
+    * bottom padding is 75px
+    * left padding is 100px
 
     padding: 20px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is 20px
+    * right padding is 20px
+    * bottom padding is 20px
+    * left padding is 20px
 
 6. What is the difference between padding and margin?
```

Comment:
:+1: Remember that the padding is a part of the HTML element and the margin is not.